idb/postgres/postgres.go

```diff
@@ -790,25 +790,41 @@ func (db *IndexerDb) yieldTxnsThreadSimple(ctx context.Context, rows pgx.Rows, r
 		var round uint64
 		var asset uint64
 		var intra int
-		var txnbytes []byte
-		var roottxnbytes []byte
+		var txn []byte
+		var roottxn []byte
 		var extraJSON []byte
```

Comment:
Since these are all the same jsonb column type, could you rename `extraJSON` to `extra`?

Reply:
done